### DISCUSSION TITLE
GitHub Desktop commit changes problem

### DISCUSSION THREAD

**Discussion by xxorianna** [community/community#50137] [Other Feature Feedback, Questions, & Ideas]
Bug

So I've been doing a coding workshop and using the GitHub Desktop app to commit my changes to the project. Everything had been going well until suddenly the app couldn't locate my local files on my hard drive where my code was located. I did not change anything, so I'm not sure what could have happened. So I was unable to work on my code. I tried cloning the repository to a different location on my hard drive but it doesn't see any changes that I've made in Visual Studio Code.

**Comment by enviGit** [comment]
### Hi Rianna!

It sounds like there may be a few potential issues going on.

- Double-check that you are opening the correct folder in VSC. If you accidentally opened the wrong folder, you may not see the changes you made in GitHub Desktop.
- Make sure that you are signed in to the correct GitHub account in GitHub Desktop. If you are not signed in, you may not be able to see your repositories.
- Check the status of your files in GitHub Desktop. Are they showing as  "**uncommitted changes**"? If so, you should be able to commit them by clicking the **commit** button.
- If none of these steps work, you may need to try reinstalling GitHub Desktop. This can sometimes help fix issues with the app not recognizing your local files.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @xxorianna ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
